src/TypeScript.UnitTests/Analyzer/EslintBridgeIssueConverterTests.cs

```diff
@@ -18,18 +18,22 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System;
 using System.Collections.Generic;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SonarLint.VisualStudio.Core.Analysis;
 using SonarLint.VisualStudio.TypeScript.Analyzer;
 using SonarLint.VisualStudio.TypeScript.EslintBridgeClient.Contract;
+using SonarLint.VisualStudio.TypeScript.Rules;
 
 namespace SonarLint.VisualStudio.TypeScript.UnitTests.Analyzer
 {
     [TestClass]
     public class EslintBridgeIssueConverterTests
     {
+        private const string TestEslintBridgeRuleId = "rule id";
```

Comment:
A bit ugly but I don't have a neat way to mock the collection and it's simpler.

Reply:
I think it's ok, but I'd change the name to better indicate that the value isn't important as it isn't checked; it just needs to be valid e.g. `ValidEsLintKey`, `AnyEslintKey`.